setup.py

```diff
@@ -43,10 +43,10 @@
 
 PACKAGE = {
     'name': 'vsc-base',
-    'version': '2.4.9',
+    'version': '2.4.9.1',
```

Comment:
that'll screw up ordering, for example in `module avail`, sure you want `2.4.10`?